data/geolocation-!cn

```diff
@@ -129,6 +130,7 @@ include:streamable
 # Forums & Social network
 include:category-forums
 include:category-social-media-!cn
+include:category-communication
```

Comment:
I did not get it.

Reply:
I tend to move "category-communication" items into "category-social-media-!cn", but that might be a breaking change